src/robot/tidypkg/transformers.py

```diff
@@ -231,9 +231,14 @@ def _remove_consecutive_separators(self, line):
 
     def _normalize_spaces(self, index, token, line_length):
         if token.type == Token.SEPARATOR:
-            spaces = self.space_count * self.indent \
-                if index == 0 else self.space_count
-            token.value = ' ' * spaces
+            if self.space_count != 'preserve':
+                spaces = self.space_count * self.indent \
+                    if index == 0 else self.space_count
+                token.value = ' ' * spaces
+            else:
+                if index == 0:
+                    token.value = ' ' * 4 * self.indent
```

Comment:
so preserve option preserve spaces but only between arguments? Let's say I have ident of 8 spaces in my test, I would assume that preserve option would not change this:

```
My Test
[Template]  Some Template
            1             2
            3             4
```

Reply:
This is due to that tidy sometimes adds a token (e.g. `END` when changing old FOR loop syntax to the new one) and it needs some spacing to not be justified to the left without whitespaces but you're right that someone may use other indentation than 4 spaces so I need to test it more and come up with an idea to use the same spacing as it is in the file.

Reply:
Ok, I did some testing and it seems that tidy works like this. It has some rules about parsing templates and some default indentation. I think it's good that it does some automatic parsing of strange things like too wide indent and it should stay like this thus I will not make changes in this area. I picked 4 spaces as a default value because there need to be some default value 😉  and I also think that it's the most popular indentation style in space separated robot files so maybe it's good to keep it like this.